src/homepageExperience/components/steps/cli/InitializeClient.tsx

```diff
@@ -104,14 +104,21 @@ export const InitializeClient: FC<OwnProps> = ({
       dispatch(createAuthorization(authorization))
       event(`firstMile.${wizardEventName}.tokens.tokenCreated`)
     }
-  }, [sortedPermissionTypes.length])
+  }, [
+    dispatch,
+    me.id,
+    org.id,
+    sortedPermissionTypes,
```

Comment:
This is likely good since `sortedPermissionTypes` is memoized, but let's talk about it.

Reply:
Talked about this in pairing - sticking with `sortedPermissionTypes.length` here may be sufficiently safe, since what we care about is whether or not the array has `length`.